### PR TITLE
Remove duplicate configuration

### PR DESCRIPTION
**Pull request by daniel-kurzynski**
# Changes

- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: vars/cloudFoundryDeploy.groovy [hunk 1]
```diff
@@ -116,10 +116,6 @@ import static com.sap.piper.Prerequisites.checkScript
      * Additional parameters passed to mta deployment command.
      */
     'mtaDeployParameters',
-    /**
-     * Defines additional extension descriptor file for deployment with the mtaDeployPlugin.
-     */
-    'mtaExtensionDescriptor',
```
Comment (o-liver) [inline vars/cloudFoundryDeploy.groovy]: Just for clarification: The nested [`cloudFoundry.mtaExtensionDescriptor`](https://github.com/alejandraferreirovidal/jenkins-library/blob/7fc25d48e8bd9a2e6e5a96b7a53d7df462478613/vars/cloudFoundryDeploy.groovy#L67) is legit while the unnested one here is deprecated?

Comment (daniel-kurzynski) [reply vars/cloudFoundryDeploy.groovy]: Well deprecated. I would more say both is possible, but the nest one will win.

Comment (daniel-kurzynski) [reply vars/cloudFoundryDeploy.groovy]: but you are right. maybe it is better the other way around. Will fix it.

Comment (daniel-kurzynski) [reply vars/cloudFoundryDeploy.groovy]: So I adopted it. I think this is also more like it is in go without having this deep structures.

**Comment by alejandraferreirovidal** [comment]
Ok, great, I don't mind how it is implemented as long as it can be used.

**Pull request merged**